experimental/css-has-pseudo/test/_browser.html

```diff
@@ -733,8 +733,8 @@
 
 			function testColor(el, color) {
 				var actual = getComputedStyle(el).color;
-				if (actual !== color) {
-					throw new Error('nested :has is allowed : div#' + el.id + '.color; expected ' + color + ' but got ' + actual);
+				if (actual === color) {
+					throw new Error('nested :has should not be allowed : div#' + el.id + '.color; expected ' + color + ' but got ' + actual);
```

Comment:
I added the test to main with the inverse check and that passes.
After the `core-web` update it fails 🎉